### PR TITLE
Changes for next major version - 4.0.0

### PR DESCRIPTION
**Pull request by kevinleturc**

### CODE REVIEW

**Comment by kevinleturc** [comment]
> Quite the diff! I didn't expect such intrication in the pipeline and code.

It's due to the okhttp upgrade to 4, imho we don't need to test with okhttp 3 anymore as 4 is now the required version.

File: nuxeo-java-client/src/main/java/org/nuxeo/client/objects/blob/Blobs.java [hunk 1]
```diff
@@ -40,27 +37,4 @@ public Blobs(List<? extends Blob> blobs) {
         super(EntityTypes.BLOBS, blobs);
     }
 
-    public List<Blob> getBlobs() {
-        return getEntries();
-    }
```
Comment (ataillefer) [inline nuxeo-java-client/src/main/java/org/nuxeo/client/objects/blob/Blobs.java:45]: Not marked as deprecated?

**Review by ataillefer — APPROVED** [review]
:muscle:

File: nuxeo-java-client/src/main/java/org/nuxeo/client/objects/blob/Blobs.java (hunk 1 above)
Comment (kevinleturc) [reply nuxeo-java-client/src/main/java/org/nuxeo/client/objects/blob/Blobs.java:45]: Indeed, this was a forgotten deprecation and as it is a getter we can found in other classes, I though it was ok to remove it.

**Pull request merged**